operator_console/main.py

```
@@ -1217,15 +1231,55 @@ def admin_observability_summary(
     @app.get("/api/admin/observability/operation-runs")
     def admin_observability_operation_runs(
         limit: int = Query(default=50),
         operation_type: str | None = Query(default=None),
         status: str | None = Query(default=None),
         services: ReadServices = Depends(get_read_services),
     ) -> JSONResponse:
+        """
+        Return a paginated list of operation runs for observability, optionally filtered by type or status.
+        
+        Parameters:
+            limit (int): Maximum number of items requested; will be clamped to the range 1–200.
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)